Monika After Story/game/script-topics.rpy

```diff
@@ -15667,3 +15667,92 @@ label monika_eating_meat:
     extend 3hub "That way I can still enjoy your cooking!"
     m 3eua "Whatever we eat, the most important thing to me is that we try to put a little thought into where our food comes from."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_glasses",
+            category=['You'],
+            prompt="Wearing glasses",
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+default persistent._mas_pm_wears_glasses = None
+
+label monika_player_glasses:
+    m 3lsb "Hey, [player], I was wondering about something lately."
+    m 1esa "Do you wear glasses, or maybe contact lens?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Do you wear glasses, or maybe contact lens?{fast}"
+        "I always wear them.":
+            jump monika_player_glasses_always
+        "I wear cosmetic glasses.":
+            jump monika_player_glasses_cosmetic
+        "I wear them as needed.":
+            jump monika_player_glasses_as_needed
+        "Nope, I don't.":
+            jump monika_player_glasses_never
+
+label monika_player_glasses_cosmetic:
+    $ persistent._mas_pm_wears_glasses = False
+    m 5ekbsa "Aw, [player], I wish I really could see your face with them on."
+    m 5hubsa "I bet you look so smart, intelligent and confident, ehehe."
+    m 1sua "Maybe I could even get myself a pair and try them out, too, ehehe."
+    if mas_isMoniAff(higher=True):
+        m 1tuu "What do you think, honey? Would they suit me well, hmm?"
+    else:
+        m 1tuu "What do you think, [player]? Would they suit me well, hmm?"
+    return
+
+label monika_player_glasses_always:
+    $ persistent._mas_pm_wears_glasses = True
+    m 2ekc "Oh, I see. You've got a poor eyesight, haven't you?"
+    m 2eka "I really hope you see better with them on."
+    m 3eub "And there's actually a bright side of wearing glasses - I'm sure you look so smart and intelligent, ehehe."
+    m 1eka "Please take good care of yourself, [player], even when it comes to something minor like this."
+    m 1esa "If you've been on your computer for a while, maybe you could consider taking a little break to let your eyes rest."
+    m 3hsa "It'll take you just a few minutes a day, but I believe it would really help."
+    m 1ekc "Although... if it's really bad, come to visit a doctor, okay?"
+    m 1eka "I really want my sweetheart to see this beautiful world just as clear~"
```

Comment:
m 1eka "I really want my sweetheart to see this beautiful world just as clear as I do~"
or
`    m 1eka "I really want my sweetheart to see this beautiful world for as long as possible~"`

It's unclear to me what "just as clear" means, so maybe one of the two options above is better?